tensorflow/python/training/experimental/loss_scale.py

```diff
@@ -196,7 +196,7 @@ def update(self, grads):
 def _is_all_finite(grads):
```

Comment:
If you want, also update the [Keras LossScale](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/mixed_precision/experimental/loss_scale.py). If you don't, I'll do it after this is submitted.

Reply:
Sounds good.  I applied the same changes to the keras loss_scale.py and loss_scale_test.py